### PR TITLE
fix: fees part 1 cleanup

### PR DESCRIPTION
**Pull request by elmutt**
- remove fees returning from buildSendTx
- remove recipients from buildSendTx
- add btc specific fee data and stub out return values

https://github.com/shapeshift/web/issues/215

We need to show fees on frontend and let them be selectable (it's all showing 0 right now) but the way we were expecting to do fees in frontend code was only applicable to eth and not the greatest (getting the fees from calling buildSendTx instead of getFees).

I will have a few additional changes to lib in upcoming prs but wanted to break it into smaller pieces.

### CODE REVIEW

**Comment by cjthompson** [comment]
Can you link to a ticket for this for context, or if there's no ticket, provide context. I don't understand the implications of the changes.

**Review by cjthompson — APPROVED** [review]
Approved but I don't have context around this change so I can't comment on correctness.

File: packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts [hunk 1]
```diff
@@ -125,34 +126,46 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
     tx: chainAdapters.BuildSendTxInput
   ): Promise<{
     txToSign: chainAdapters.ChainTxType<ChainTypes.Bitcoin>
-    estimatedFees: chainAdapters.FeeDataEstimate<ChainTypes.Bitcoin>
   }> {
     try {
       const {
         value,
         to,
-        recipients,
```
Comment (0xdef1cafe) [inline packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts:134]: remove this from the type too

File: packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts [hunk 2]
```diff
@@ -125,34 +126,46 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
...
+          'BitcoinChainAdapter: signTransaction wallet does not support signing btc txs'
+        )
+
+      const from = await wallet.btcGetAddress({
+        addressNList,
+        coin: this.coinName,
+        scriptType
+      })
+
+      if (!from) throw new Error('from undefined')
```
Comment (0xdef1cafe) [inline packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts:165]:
```suggestion
      if (!from) throw new Error('BitcoinChainAdapter: from undefined')
```

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**